chike-chike17/week-2/js-racer-game-lab-master/starter-code/app.js

```diff
@@ -0,0 +1,67 @@
+let intervalCode;
+let windowPosition = window.innerWidth -20;
+let startGo;
+let go;
+
+let mario = document.getElementsByClassName('mario');
+mario = mario[0];
+mario.style.left = "0px";
+
+let bowser = document.getElementsByClassName('bowser');
+bowser = bowser[0];
+bowser.style.left = "-40px";
+
+let luidgi = document.getElementsByClassName('luidgi');
+luidgi = luidgi[0];
+luidgi.style.left = "0px";
+
+
+let start = document.getElementById('start');
+let bowserPosition;
+
+
+let playStop = function () {
+  intervalCode = setInterval(()=> {
+        bowser.style.left = parseInt(bowser.style.left) + 20 + 'px';
+        bowserPosition = bowser.offsetWidth + parseInt(bowser.style.left) -5;
+        if (bowserPosition > windowPosition ) {
+           alert('Bowser Won!!!!');
+        }
+      }, 100);
+}
+
+start.addEventListener('click', function () {
+  playStop();
+});
+
+let goStop = document.getElementById("go-stop");
+
+goStop.addEventListener("click", function () {
+  if (!go){
+    clearInterval(intervalCode);
+    go = !go;
+  } else {
+    playStop();
+    go = !go;
+  }
+});
+
+
```

Comment:
I like the logic you have going on this page. I would add more notation for yourself in case you need to come back and rework these functions.